extensions/default/src/DicomWebProxyDataSource/index.ts

```diff
@@ -46,7 +46,7 @@ function createDicomWebProxyApi(dicomWebProxyConfig, servicesManager: AppTypes.S
       },
     },
     retrieve: {
-      directURL: (...args) => dicomWebDelegate.retrieve.directURL(...args),
```

Comment:
Please revert this change, and add getDataURL.  There is no need to make it "async" as the return from the nested instance is already a promise and that is all async means (plus the handlign to allow await)  Doing it unnecessarily adds quite a bit of overhead.